src/undoredo.cpp

```diff
@@ -48,6 +48,7 @@ void SolveSpaceUI::PushFromCurrentOnto(UndoStack *uk) {
     UndoState *ut = &(uk->d[uk->write]);
     *ut = {};
     ut->group.ReserveMore(SK.group.n);
+    // Not using range-for in most of the below because it's unclear if sizes are changed.
```

Comment:
The size of anything in `SK` is not changed.

Reply:
Updated in the new push.